wire-ios-data-model/Source/ManagedObjectContext/CoreDataStack+Backup.swift

```diff
@@ -169,13 +169,20 @@ extension CoreDataStack {
         dispatchGroup: ZMSDispatchGroup? = nil,
         completion: @escaping ((Result<URL>) -> Void)
     ) {
+        guard let activity = BackgroundActivityFactory.shared.startBackgroundActivity(withName: "import backup") else {
+            completion(.failure(CoreDataStackError.noDatabaseActivity))
+            return
+        }
         importLocalStorage(
             accountIdentifier: accountIdentifier,
             from: backupDirectory,
             applicationContainer: applicationContainer,
             dispatchGroup: dispatchGroup,
             messagingMigrator: CoreDataMessagingMigrator(isInMemoryStore: false),
-            completion: completion
+            completion: { result in
+                completion(result)
```

Comment:
same here, maybe good to add a log here if the outcome is not logged on completion

Reply:
performRestore